lars-reimann/api-editor
# fix: crash when importing generated annotation file

Closes #590.

### Summary of Changes

Fix a crash when a generated annotation file (without `currentUserAction`) is imported.

### Testing Instructions

1. Open the GUI in the browser
2. File > Import Annotations
3. Choose an annotions file, e.g. `data/annotations/sklearn__annotations.json`
4. Should not crash anymore
